ibis/backends/tests/test_union.py

```diff
@@ -23,7 +23,7 @@ def test_union(backend, alltypes, df, distinct):
         param(
             True,
             marks=pytest.mark.notimpl(
-                ["postgres", "pyspark"],
+                ["duckdb", "postgres", "pyspark"],
```

Comment:
Hm, this seems like it should be supported out of the box.

Reply:
I think the only bit that isn't supported here is guaranteed ordering with a distinct